File: solutions/2089-maximum-matrix-sum/solution.java
class Solution {
    public long maxMatrixSum(int[][] matrix) {
        int neg=0,a;
        long sum=0,min=1000000;

        for(int i=0;i<matrix.length;i++)
        {
            for(int j=0;j<matrix[i].length;j++)
            {
                if(matrix[i][j]<0) neg++;
                a=Math.abs(matrix[i][j]);
                sum+=a;
                min=Math.min(min,a);
            }
        }
        System.out.print(sum);
        if(neg%2==0)
            return sum;
        return sum-min*2;
    }
}

File: solutions/2089-maximum-matrix-sum/solution.py
class Solution(object):
    def maxMatrixSum(self, matrix):
        """
        :type matrix: List[List[int]]
        :rtype: int
        """
        neg=0
        m=1000000
        s=0
        for i in matrix:
            for j in i:
                if j<0:
                    neg+=1
                a=abs(j)
                s+=a
                m=min(m,a)
        if neg%2==0:
            return s
        else:
            return s-m*2
